network-api/networkapi/wagtailpages/pagemodels/blog/blog.py

```diff
@@ -311,3 +316,7 @@ def get_meta_description(self):
                 return truncatechars(text, 153)
 
         return super().get_meta_description()
+
+    def save(self, **kwargs):
+        self.clean()
+        return super(BlogPage, self).save(**kwargs)
```

Comment:
**Question** Is this necessary? This should be handled by Wagtail already. 

I know it's a bit of an odd behavior of Django that it does not automatically validate models on save, but mixing these things now might cause other issues. Typically the call to the `clean` or rather `full_clean` will be done in the view of the form.

Reply:
No longer using! Going to mark this as resolved